### ISSUE TITLE
WCAG 2.2 Violation - There are links with non-descriptive accessible names present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#9]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1. 

### Instances

- #35
